### PR TITLE
feat: deleted "User Subscriptions" section

### PR DESCRIPTION
**Pull request by FredericEspiau**
### Description of the Change

As this service was discontinued, the section was deleted and the menu item too.

### CODE REVIEW

File: client/app/account/user/user.module.js [hunk 1]
```diff
@@ -27,10 +27,23 @@ angular
             });
 
             if (target === "EU") {
-                $stateProvider.state("app.account.useraccount.subscriptions", {
-                    url: "/subscriptions",
-                    templateUrl: `${baseUrl}subscriptions/user-subscriptions.html`,
-                    controller: "UserAccount.controllers.Subscriptions"
+                $stateProvider.state("app.account.useraccount.subContacts", {
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply client/app/account/user/user.module.js]: I did rebase before pushing though

**Comment by FredericEspiau** [comment]
Fixed @frenautvh

File: client/app/components/user/session/user-session.service.js [hunk 2]
```diff
@@ -370,9 +370,6 @@ class SessionService {
                         state: "app.account.useraccount.emails"
                     },
                     (this.constants.target === "EU") && {
```
Comment (frenautvh) [inline client/app/components/user/session/user-session.service.js]: you forgot to remove the conditional

Comment (FredericEspiau) [reply client/app/components/user/session/user-session.service.js]: Did not understand how I worked, I will remove it

**Comment by FredericEspiau** [comment]
@frenautvh FIXED

**Pull request merged**